lib/wasix/src/fs/mod.rs

```diff
@@ -930,7 +930,8 @@ impl WasiFs {
                                 let (pre_open_dir_fd, relative_path) = if link_value.is_relative() {
                                     self.path_into_pre_open_and_relative_path(&file)?
                                 } else {
-                                    unimplemented!("Absolute symlinks are not yet supported");
+                                    tracing::error!("Absolute symlinks are not yet supported");
```

Comment:
We don't wan't logging in the `virtual-fs` crate, that should be kept to usage sites like `wasix`.